### DISCUSSION TITLE
How to use sqlite extension with sqlite

### DISCUSSION THREAD

**Discussion by spa5k** [prisma/prisma#17016] [Q&A]
So, I want to run this extension with my sqlite database - https://github.com/phiresky/sqlite-zstd but i need to add it into the instance, is there any way to achieve it??

**Comment by pantharshit00** [comment]
Hello @spa5k,

Looks like the sqlite install that we ship doesn't have extension loading enabled so you will need to create a database manually and load extension there in order to use this. I will recommend you to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) about sqlite extension usage.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
